Makefile

```diff
@@ -126,14 +126,15 @@ release/clean:
 	rm -rf release
 
 .PHONY: acceptance
-acceptance: release/clean docker-build docker-push release
+acceptance: release/clean docker-build release
 	ACCEPTANCE_TEST_SECRET_TYPE=token make acceptance/kind acceptance/setup acceptance/tests acceptance/teardown
 	ACCEPTANCE_TEST_SECRET_TYPE=app make acceptance/kind acceptance/setup acceptance/tests acceptance/teardown
 	ACCEPTANCE_TEST_DEPLOYMENT_TOOL=helm ACCEPTANCE_TEST_SECRET_TYPE=token make acceptance/kind acceptance/setup acceptance/tests acceptance/teardown
 	ACCEPTANCE_TEST_DEPLOYMENT_TOOL=helm ACCEPTANCE_TEST_SECRET_TYPE=app make acceptance/kind acceptance/setup acceptance/tests acceptance/teardown
 
 acceptance/kind:
 	kind create cluster --name acceptance
+	kind load docker-image ${NAME}:${VERSION} --name acceptance
```

Comment:
I have not tried myself yet but probably- `kind load`ing the runner image as well would further speed up the test run?

Reply:
@mumoshu That is one of the other images i was talking about to cache on the host :smiley: 
So i would probably download them to the host docker once, and then kind load on the four clusters.
I also think it would make sense to have the change you told me in deploy.sh should be done in the repo, if this is an issue for everyone.
Should i just add that to my next PR?

Reply:
@ToMe25 Ah gotcha! Yeah, adding that to your next PR would be very much appreciated 👍